### PR TITLE
ci-3011 - GAR support 

### PR DESCRIPTION
**Pull request by eoinmcafee00**

### CODE REVIEW

File: cmd/drone-gcr/main.go [hunk 1]
```diff
@@ -16,45 +18,80 @@ import (
...
+
+	registry := getenv("PLUGIN_REGISTRY")
 	if registry == "" {
-		registry = "gcr.io"
+		switch registryType {
+		case "GCR":
+			registry = "gcr.io"
+		case "GAR":
+			if location == "" {
+				logrus.Fatalf("Error: For GAR, LOCATION must be set")
```
Comment (tphoney) [inline cmd/drone-gcr/main.go:65]: can you change the error message to, 

"Error: For REGISTRY_TYPE of GAR, LOCATION must be set" 

and squash your commits.

**Review by tphoney — CHANGES_REQUESTED** [review]

File: cmd/drone-gcr/main.go (hunk 1 above)
Comment (eoinmcafee00) [reply cmd/drone-gcr/main.go]: done and I'll squash on merge :)

Comment (tphoney) [reply cmd/drone-gcr/main.go]: @eoinmcafee00 you didnt push. no change yet

Comment (eoinmcafee00) [reply cmd/drone-gcr/main.go]: https://github.com/drone-plugins/drone-docker/pull/404/files#diff-c2b522394f8c3e1528f8508b7d9fbb24b34093aca68a56b4cf0bea2b85bf5d98R65

**Pull request merged**